### ISSUE TITLE
structfromnt lacks flexibility

### ISSUE THREAD

**Issue by matbesancon** [Wikunia/Bonobo.jl#37]
It does require all structs of the node to be passed as info, it is not super idiomatic since it does not leverage multiple constructors for instance.
I would remove it in favour of dispatch IMO

**Comment by Wikunia** [comment]
Yeah it's definitely not ideal. Will have a look at it again over the weekend.
